src/randgraphs.jl

```diff
@@ -37,7 +41,7 @@ Note also that Erdős–Rényi graphs may be generated quickly using `erdos_reny
 or the  `Graph(nv, ne)` constructor, which randomly select `ne` edges among all the potential
 edges.
 """
-function erdos_renyi(n::Integer, p::Real; is_directed=false, seed::Integer=-1)
+function erdos_renyi(n::Integer, p::AbstractFloat; is_directed=false, seed::Integer=-1)
```

Comment:
I wanted to enforce `p` to be a floating point number, and for example an integer, for which we have `Integer<:Reals`. But since I hadn't considered Rational type and since integers are already captured by the other signature of erdos_renyi I'm reverting this change (I didn't even remember doing that, the change I was considering before was is_directed -> directed)